### ISSUE TITLE
Pipfile not found error when use a requirments.txt 

### ISSUE THREAD

**Issue by harshad16** [thoth-station/s2i-minimal-notebook#27]
**Describe the bug**
Trying to build a new image with s2i-minimal-notebook as the base image.Using requirments.txt for package install through error during build time. 

`File 'Pipfile' not found in '/opt/app-root/src' or any parent directory`

**Expected behavior**
support new image build either Pipfile or requirments.txt

**Comment by fridex** [comment]
I see `quay.io/thoth-station/s2i-thoth-ubi8-py36:latest` as a base so micropipenv should be used (also judging based on the error message). Are files present in `/opt/app-root/src`? Pipenv files take precedence over requirement files.

**Comment by fridex** [comment]
Checking our base images during other work, it looks like we keep Pipfile.lock in the base image which is not good:
```
$ podman run --entrypoint bash -it --rm quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.23.2 -c ls
Pipfile.lock
```

**Comment by harshad16** [comment]
thanks @fridex  for the valuable suggestion.

**Comment by harshad16** [comment]
Fixed this issue in v0.0.7

**Comment by fridex** [comment]
After thinking about this a bit more, I think the issue here is the actual Thoth's s2i build process. As we use micropipenv to install the software in Thoth s2i itself, it will mark down the lock file which is not desired in this case. I'll open a PR to fix this as this will affect all the users that use Thoth's s2i base images and requirements.txt files as discussed yesterday with you. Sorry for the confusion here, it was not clear to me where the Pipfile.lock comes from.
